syncthing/README.md

```diff
@@ -0,0 +1,62 @@
+# Agent Check: Syncthing
+
+## Overview
+
+This check monitors [Syncthing][1].
```

Comment:
Can you provide more detail here? Something like:

"Syncthing synchronizes files between two or more computers in real time. This integration allows you to monitor [Syncthing][1] using Datadog."

Reply:
Fixed!